input/intro-notes/StructureDefinition-au-practitioner-intro.md

```diff
@@ -5,3 +5,5 @@
 - A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification` 
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
+- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
```

Comment:
Missing the implementation guidance that was present. Put back but change it to remove the reference to the examples.

Reply:
Reinstated guidance & removed links to examples.

Reply:
I see it reinstated but have lost the following point that needs to be present:
- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).

Reply:
Reinstated lost point.